docs/src/main/asciidoc/update-to-quarkus-3.adoc

```diff
@@ -0,0 +1,55 @@
+////
+This document is maintained in the main Quarkus repository, and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="update-projects-to-quarkus-3-automatically-howto"]
+= Update projects to Quarkus 3 automatically
+include::_attributes.adoc[]
+:categories: core
+:extension-status: "experimental"
+:summary: Update projects from Quarkus 2.13 and later to Quarkus 3.
```

Comment:
The guide isn't just Quarkus 2.13 and later Quarkus 3 - the update tooling will also work for Quarkus 3.0-3.x's

Reply:
scratch that - now having  read it all i see this is specifically for quarkus 3 migration. not a generic explanation of the update toolling.

Reply:
Thank  you, Max.

Reply:
Thanks for your thoughts on this.